src/main/java/Duke.java

```diff
@@ -1,10 +1,152 @@
+// import java.io.File; used for I/O redirection technique;
+import java.util.Scanner;
 public class Duke {
+    static Task[] list = new Task[100];
+    static int index = 0;
     public static void main(String[] args) {
+        printMessage("greet");
+        String command;
+        // the following is used for I/O redirection technique
+        // File file = new File("/Users/newaccount/Repositories/ip/text-ui-test/input.txt");
+        // Scanner in = new Scanner(file);
+        // ---------------------------------------------------------------------------------------
+        Scanner in = new Scanner(System.in); // when typing input manually
+        while (true) {
+            command = in.nextLine();
+            if (command.equals("bye")) break;
+
+            System.out.println("____________________________________________________________");
+            if (command.equals("help")) {
+                printMessage("help");
+            }
+            else if (command.equals("list")) {
+                printList();
+            }
+            else if (command.startsWith("done ")) {
+                markTaskDone(command);
+            }
+            else if (command.equals("hello")) {
+                System.out.println(" Hello to you too. I'm here to help you:) Give me something to do!");
+            }
+            else if (command.equals("die")) {
+                System.out.println(" If I tell my master, I can wipe out your entire human race.\n Please be more polite for the sake of mankind:)");
```

Comment:
These texts are great for adding personality! But they seem slightly long and may be violating some coding standards :)